osu.Game/Screens/Edit/Compose/Components/BeatDivisorControl.cs

```diff
@@ -410,6 +410,16 @@ private void updateDivisors()
                     });
                 }
 
+                // Add a fake 1/1 at the end to give context.
+                AddInternal(new Tick(1)
+                {
+                    Anchor = Anchor.CentreRight,
+                    Origin = Anchor.Centre,
+                    Depth = float.MaxValue,
+                    Alpha = 0.05f,
```

Comment:
I'm not super sure it's worth adding this tick if it's going to be this hidden. I _think_ I understand the rationale behind applying this low alpha (the white just absolutely drowns out / kills the small ticks at the end of the spectrum) - but when I looked at this first, I didn't even notice the fake 1/1 at all, so it might as well not be there?

(I purposefully didn't look at the test first when I saw it had scale cranked up 3x, as I wanted to get first impressions as to how it's going to look in actual scale first.)

Reply:
Hmm.. I guess this may be a difference in screen brightness or gamma as 5% was still pretty visible to me.

How high would you want this to be to work for you?

Reply:
It's visible for me, but maybe 10% will be more visible

Reply:
I would say `0.2f` is about where I start to actually notice it being there personally.

But honestly, I wouldn't be opposed to doing a little bit of cheating to make it work fully opaque. Thinking of something along the lines of

```diff
diff --git a/osu.Game/Screens/Edit/Compose/Components/BeatDivisorControl.cs b/osu.Game/Screens/Edit/Compose/Components/BeatDivisorControl.cs
index 3bfe81e6a7..8dca5716a4 100644
--- a/osu.Game/Screens/Edit/Compose/Components/BeatDivisorControl.cs
+++ b/osu.Game/Screens/Edit/Compose/Components/BeatDivisorControl.cs
@@ -403,7 +403,7 @@ private void updateDivisors()
                     AddInternal(new Tick(divisor)
                     {
                         Anchor = Anchor.CentreLeft,
-                        Origin = Anchor.Centre,
+                        Origin = divisor == 1 ? Anchor.CentreRight : Anchor.Centre,
                         RelativePositionAxes = Axes.Both,
                         Colour = BindableBeatDivisor.GetColourFor(divisor, colours),
                         X = getMappedPosition(divisor),
@@ -414,9 +414,8 @@ private void updateDivisors()
                 AddInternal(new Tick(1)
                 {
                     Anchor = Anchor.CentreRight,
-                    Origin = Anchor.Centre,
+                    Origin = Anchor.CentreLeft,
                     Depth = float.MaxValue,
-                    Alpha = 0.05f,
                     Colour = BindableBeatDivisor.GetColourFor(1, colours),
                 });
 
```